ToDo/ToDo/DTO/ErrorResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace ToDo.DTO
```

Comment:
```suggestion
﻿namespace ToDo.DTOs
```
tên folder baoh cx là số nhiều nhé